src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -0,0 +1,428 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { LitElement, html, customElement, property } from 'lit-element';
+import * as MicrosoftGraph from '@microsoft/microsoft-graph-types';
+
+import { Providers } from '../../Providers';
+import { ProviderState } from '../../providers/IProvider';
+import { styles } from './mgt-people-picker-css';
+
+import '../mgt-person/mgt-person';
+import '../../styles/fabric-icon-font';
+import { MgtTemplatedComponent } from '../templatedComponent';
+import { MgtPersonDetails, MgtPerson } from '../mgt-person/mgt-person';
+
+@customElement('mgt-people-picker')
+export class MgtPicker extends MgtTemplatedComponent {
+  private _firstUpdated = false;
+  private groupPeople;
+  private isLoading = false;
+
+  @property({
```

Comment:
Add comment?  Does this just represent the current filtered list of people?  Can I provide my own list of people I want to have as options for the search instead of calling the graph?  (i.e. I called the graph separately elsewhere and want to provide those results as options for the drop-down here.)

Reply:
Added comments, and cleaned up some of the extra variables:

https://github.com/microsoftgraph/microsoft-graph-toolkit/commit/b3dd155381eee3668e477863cb553be8f16021fe